ISISComputingGroup/ibex_gui
# Ticket898 beam in wrong place

To test:
- Switch to a synoptic with many components and groupings (e.g. larmor)
- Confirm that all components are in the right place and that controls properly line up
